python/cugraph/cugraph/tests/centrality/test_batch_betweenness_centrality_mg.py

```diff
@@ -19,23 +19,23 @@
 from cugraph.dask.common.mg_utils import is_single_gpu
 from cugraph.experimental.datasets import karate
 
+from test_betweenness_centrality import (
+    calc_betweenness_centrality,
+    compare_scores,
+)
+
 # Get parameters from standard betwenness_centrality_test
 # As tests directory is not a module, we need to add it to the path
 # FIXME: Test must be reworked to import from 'cugraph.testing' instead of
 # importing from other tests
```

Comment:
This comment should probably be removed now.

Reply:
dropped